### PR TITLE
LPS-202231 Change Configuration view for Wiki

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-202231)

## Changes

Change the configuration for Bookmarks because it should be open in a new page instead of a modal.

## Test Scenario 1

1. Go to Wiki
2. Click on the kebab menu in the top right and choose "configuration"
3. Navigate to the sections to check the behaviour and styles

https://github.com/liferay-echo/liferay-portal/assets/93203423/eda6ec0b-7b0b-4ad1-8174-28161dfa7b54

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki_admin/configuration_browse.jsp [hunk 1]
```diff
@@ -36,5 +42,112 @@ WikiConfigurationDisplayContext wikiConfigurationDisplayContext = new WikiConfig
...
+			<liferay-portlet:actionURL portletConfiguration="<%= true %>" var="configurationActionURL">
+				<portlet:param name="navigation" value="<%= wikiConfigurationDisplayContext.getNavigation() %>" />
+				<portlet:param name="serviceName" value="<%= WikiConstants.SERVICE_NAME %>" />
+				<portlet:param name="settingsScope" value="group" />
+			</liferay-portlet:actionURL>
+
+			<aui:form action="<%= configurationActionURL %>" method="post" name="fm">
+				<clay:sheet
+					cssClass="c-mb-4 c-mt-4 c-p-0"
+					size="full"
```
Comment (sandrodw3) [inline modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki_admin/configuration_browse.jsp:62]: I'm not sure if this is needed 😛

**Comment by jkappler** [comment]
@BarbaraCabrera, looks good from backend side ❤️

**Comment by BarbaraCabrera** [comment]
see https://github.com/liferay-lima/liferay-portal/pull/5009

**Pull request closed without merging**